DirectXTex/BCDirectCompute.cpp

```diff
@@ -20,13 +20,29 @@ using Microsoft::WRL::ComPtr;
 
 namespace
 {
-    #include "Shaders\Compiled\BC7Encode_EncodeBlockCS.inc"
-    #include "Shaders\Compiled\BC7Encode_TryMode02CS.inc"
-    #include "Shaders\Compiled\BC7Encode_TryMode137CS.inc"
-    #include "Shaders\Compiled\BC7Encode_TryMode456CS.inc"
-    #include "Shaders\Compiled\BC6HEncode_EncodeBlockCS.inc"
-    #include "Shaders\Compiled\BC6HEncode_TryModeG10CS.inc"
-    #include "Shaders\Compiled\BC6HEncode_TryModeLE10CS.inc"
+    namespace cs5
+    {
+        #include "Shaders\Compiled\BC7Encode_EncodeBlockCS.inc"
+        #include "Shaders\Compiled\BC7Encode_TryMode02CS.inc"
+        #include "Shaders\Compiled\BC7Encode_TryMode137CS.inc"
+        #include "Shaders\Compiled\BC7Encode_TryMode456CS.inc"
+        #include "Shaders\Compiled\BC6HEncode_EncodeBlockCS.inc"
+        #include "Shaders\Compiled\BC6HEncode_TryModeG10CS.inc"
+        #include "Shaders\Compiled\BC6HEncode_TryModeLE10CS.inc"
+    }
+
+#if !defined(_XBOX_ONE) || !defined(_TITLE)
```

Comment:
This logic is just to keep the Xbox One static library a little smaller since it will always use the CS 5.0 version.